packages/contract-tests/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@odh-dashboard/contract-tests",
+  "version": "1.0.0",
```

Comment:
Not a big deal, but we aren't versioning our monorepo packages. Just for consistency with the others.

```suggestion
  "version": "0.0.0",
```